packages/x-date-pickers-pro/src/DesktopDateTimeRangePicker/DesktopDateTimeRangePicker.tsx

```diff
@@ -88,25 +65,23 @@ const rendererInterceptor = function rendererInterceptor<
   const isTimeViewActive = isInternalTimeView(popperView);
   return (
     <React.Fragment>
-      {inViewRenderers.day?.({
+      {viewRenderers.day?.({
         ...rendererProps,
         availableRangePositions: [rangePosition],
         view: !isTimeViewActive ? popperView : 'day',
         views: rendererProps.views.filter(isDatePickerView),
         sx: [{ gridColumn: 1 }, ...finalProps.sx],
-      })}
+      } as any)}
```

Comment:
Not a big fan of the ˋanyˋ cast here, but I guess there is no other way

Reply:
My latest changes seems to make the `as any` not needed anymore :ok_hand: